demos/demo-react-ts/package.json

```diff
@@ -6,7 +6,8 @@
   "scripts": {
     "build": "webpack",
     "build:gh": "npm ci && npm run build",
-    "start": "npm run build && webpack-dev-server --open",
+    "serve": "webpack serve",
```

Comment:
Updated script with latest documentation from https://github.com/webpack/webpack-dev-server

Reply:
Curious, why is the command for minimal-js different?

Reply:
I was deciding where to put the `--open` script 😅 I think they should be in the `start` command.